validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -28,7 +28,13 @@
     "  * 78 million objects  \n",
     "  * 48 million objects with i-band SNR > 5\n",
     "\n",
-    "Loading the entire catalog takes ~15 minutes, depending on memory load on the JupyterHub node.  We've thus entered an era where it will be common to only look at subsamples of the full DC2 datasets, whether that's considering just one tract, or taking a subsample of the full catalog."
+    "Loading the entire catalog takes ~15 minutes, depending on memory load on the JupyterHub node.  We've thus entered an era where it will be common to only look at subsamples of the full DC2 datasets, whether that's considering just one tract, or taking a subsample of the full catalog.\n",
```

Comment:
Does this memory get cached? I'm wondering if the load time is much quicker the second time you load the catalog. As a DES person, we are working with much larger catalogs and I still analyze the full data set, so I'm not sure that 78M objects is really the "beginning of an era".

Reply:
The `math` import seems a bit unnecessary with the `numpy` import.

Reply:
and I think I see the answer to my caching question in cell 8 `'use_cache': False`.

Reply:
The load time statement here is outdated I think, with parquet files the time should be much shorter, no? And the `use_cache` option is no longer relevant for parquet files.

Reply:
Oh, I meant local "era", I wasn't trying to be more grandiose.

Reply:
`math.log10` on scalars is several times faster than `numpy.log10`
I haven't explicited performance tested `math.ceil`.  Nor have I explictly tested the performance of the lambda written with `math.ceil(...math.log10(...))` vs. `np.ceil(...np.log10(...))`.  It's quite possible that the disk I/O time is dominant.

But that's why I used it.

Reply:
Our effective storage I/O performance on NERSC is substantially worse than is possible from a direct disk on a computer.  We haven't fully investigated, or worked with NERSC to investigate.  Performance to/from scratch is better, as is to/from the burst buffer, but those aren't for permanent storage.

Reply:
@yymao Yes, the Parquet files are better.  But it still takes 15 minutes to load the entire catalog.